ports/libvpx/portfile.cmake

```
@@ -308,8 +311,13 @@ else()
             LOGNAME install-${TARGET_TRIPLET}-dbg
         )
 
-        file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
-        file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
+        if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
+            file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+            # This file only exists for static builds; guard the removal.
+            if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+                file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
+            endif()
+        endif()
```

Removal commands don't complain about absent files. Keep it simple.
```suggestion
        file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
        file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
```